packages/expo-updates/ios/EXUpdates/Update/Update.swift

```diff
@@ -135,6 +135,16 @@ public class Update: NSObject {
     config: UpdatesConfig,
     database: UpdatesDatabase
   ) throws -> Update {
+    // Sanity check for remotely getting a manifest through a tunnel
+    // from a dev-server, where manifest is modern but no protocol header
```

Comment:
according to the protocol specification the header is required. what is causing there not to be a header?

Reply:
This is not actually an update server, it is a dev server being accessed through a tunnel. Example URL:

```
exp://jqjnovw.douglowderexpo.8081.exp.direct
```

Reply:
Not sure I follow. The server at the other end of the tunnel is an update server, correct? A tunnel should transfer headers as well.

Reply:
The server at the other end of the tunnel is a development packager.